File: Day 5/java/TaskOne.java
import java.util.Scanner;
public class TaskOne{
public static void main (String[] args){

Scanner input = new Scanner(System.in);

System.out.print("Enter a card number ");
String cardNumber = input.nextLine();


boolean result = TaskOne.cardNumber(cardNumber);

System.out.print(result);
}


public static boolean cardNumber(String cardNumber){

char firstDigit = cardNumber.charAt(0);

int lengthOfCard = cardNumber.length();

if (firstDigit == '4' && firstDigit == '5' && firstDigit == '6' && lengthOfCard == 16 ){
 return true;
}
else if (lengthOfCard < 16 && lengthOfCard > 16 ){
 
}return false;



}
}

File: Day 5/JS/TaskOne.js
function creditCardValidator(cardNumber){

let firstDigit = cardNumber.charAt(0);

let lengthOfCard = cardNumber.length

if (firstDigit == '4' && firstDigit == '5' && firstDigit == '6' && lengthOfCard == 16 ){
 return true;
}
else if (length < 16 && length > 16){
 
}return false;
}

let value = creditCardValidator(45555555555555555)

console.log(creditCardValidator(value))
